demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1069,6 +1073,71 @@ export class DemoMeetingApp
       });
     });
 
+    const languageIdentificationCb = document.getElementById('identify-language-checkbox') as HTMLInputElement;
+    languageIdentificationCb.addEventListener('click', () => {
+      (document.getElementById('language-options').classList.toggle('hidden', !languageIdentificationCb.checked));
+      (document.getElementById('preferred-language').classList.toggle('hidden', !languageIdentificationCb.checked));
+      (document.getElementById('transcribe-language')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('content-identification-checkbox')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('content-redaction-checkbox')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('custom-language-model-checkbox')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('transcribe-entity')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+      (document.getElementById('language-model-input-text')  as HTMLInputElement).disabled = languageIdentificationCb.checked;
+    });
+
+    const languageOptionsDropDown = document.getElementById('language-options') as HTMLInputElement;
+    languageOptionsDropDown.addEventListener('click', (event) => {
```

Comment:
This function is getting very unwieldy. Consider move this callback logic to a separate function (e.g. `languageOptionsDropDownClickHandler`) and additionally adding a block comment explaining the intention of the code (i.e. select multiple but one from each group).

Reply:
Refactored in the latest commit.